.github/workflows/test.yml

```diff
@@ -144,6 +144,7 @@ jobs:
           wait-on: 'http://localhost:3000'
           browser: electron
           group: e2e
+          spec: ${{ github.ref_name == 'releases/staging' && 'cypress/{e2e,staging}/**/*.test.ts' || 'cypress/e2e/**/*.test.ts' }}
```

Comment:
do you need this second `'cypress/e2e/**/*.test.ts'` if the regex `cypress/{e2e,staging}/**/*.test.ts` already includes it?

Reply:
I don't but I err on the side of specificity